_integration_tests/test/empty_activities/get_waiting_for_correlation.js

```diff
@@ -3,9 +3,9 @@
 const should = require('should');
 const uuid = require('node-uuid');
 
-const {TestFixtureProvider, ProcessInstanceHandler} = require('../../dist/commonjs/');
+const {TestFixtureProvider, ProcessInstanceHandler} = require('../../dist/commonjs');
 
-describe('Management API: GetEmptyActivitiesForCorrelation', () => {
+describe('ManagementAPI: GetEmptyActivitiesForCorrelation', () => {
```

Comment:
Did you remove the spacing on purpose? Because in the other files is a spacing between Management and API.
```suggestion
describe('Management API: GetEmptyActivitiesForCorrelation', () => {
```

Reply:
Not really. Just a `copy pasta` thing.

Reply:
Now its `ManagementAPI` everywhere. 😉